src/main/java/core/basesyntax/csvservice/Reader.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.csvservice;
+
+import core.basesyntax.Transaction;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+public class Reader {
+    public static List<Transaction> read(String fileName) {
```

Comment:
in general it's recommend to avoid static methods, cause there might appear difficulties, when your want to have your class implement some interfaces ( you need several reader implementations) or with more complex testings, when you wanna "mock" some part of code (it's a bit out of scope, you will learn about it in future). But is this method has only utility purpose and won't have any variables(state) i think it's not critical, just keep that in mind🙂